Makefile

```diff
@@ -19,7 +19,7 @@ endif
 .RECIPEPREFIX =
 
 IMAGE_REGISTRY   ?= quay.io/opendatahub/workbench-images
-RELEASE	 		 ?= 2024b
+RELEASE	 		 ?= 2025a
```

Comment:
Shall I review also our ocp-ci jobs? Do we e.g. care about this: https://github.com/openshift/release/blob/master/ci-operator/config/opendatahub-io/notebooks/.config.prowgen#L8 ?